### PR TITLE
Add RNs for 0.1 version

### PR DESCRIPTION
**Pull request by nataliasitko**
Changes proposed in this pull request:

- Add RNs for version 0.1

### CODE REVIEW

**Review by kolodziejczak — DISMISSED** [review]

**Review by barchw — COMMENTED** [review]
File name should follow semantic versioning `0.1.0.md`

**Comment by kolodziejczak** [comment]
Yeah, sorry for misinforming. It would still pass with the wrong format so maybe we should provide some format validation to the workflow scripts?

**Comment by barchw** [comment]
> Yeah, sorry for misinforming. It would still pass with the wrong format so maybe we should provide some format validation to the workflow scripts?

Release will fail if set to `0.1.0`. I think we cannot really validate the naming of RN file before, unless it is a separate workflow triggered on creation e.g. linting

File: docs/release-notes/0.1.0.md [hunk 1]
```diff
@@ -0,0 +1 @@
+We are pleased to present the preliminary version of the Istio module. This release is primarily meant for internal testing purposes. The official release is expected to be available in about one week.
```
Comment (mmitoraj) [inline docs/release-notes/0.1.0.md:1]:
```suggestion
We are pleased to present the preliminary version of the Istio module. This release is primarily meant for internal testing purposes. Expect the official release in about one week.
```

**Review by mmitoraj — APPROVED** [review]

**Pull request merged**